share/spack/qa/run-flake8

```diff
@@ -21,9 +21,12 @@ fi
 changed=$(git diff --name-only develop... | grep '.py$')
 
 # Exempt url lines in changed packages from overlong line errors.
+# Exempt functions defined with '@when' decorator from redefinition errors.
```

Comment:
I would move these comments next to the line they represent now that there are multiple.